PROCESSORS.md

```diff
@@ -550,18 +550,18 @@ In the list below, the names of required properties appear in bold. Any other pr
 
 ### Description
 
-Runs an operating system command specified by the user and writes the output of that command to a FlowFile. If the command is expected to be long-running,the Processor can output the partial data on a specified interval. When this option is used, the output is expected to be in textual format,as it typically does not make sense to split binary data on arbitrary time-based intervals.
+Runs an operating system command specified by the user and writes the output of that command to a FlowFile. If the command is expected to be long-running,the Processor can output the partial data on a specified interval. When this option is used, the output is expected to be in textual format,as it typically does not make sense to split binary data on arbitrary time-based intervals. This processor is not available on Windows systems.
```

Comment:
should this new line be added to the `ExecuteProcess::Description`?

Reply:
Updated in b8a9a3297a4c6e56c02a1b5767f13596ddf7b853